src/main/resources/changesets/changelog-consent-100.0.xml

```diff
@@ -0,0 +1,17 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="100.0" author="qhoque">
+        <addColumn tableName="email_entity">
+            <column name="sendgrid_response" type="text"/>
+            <column name="sendgrid_status" type="integer" defaultValue="null"/>
+            // todo: default value of date_sent for existing records
+            <column name="create_date" type="date" defaultValue="date_sent"??/>
```

Comment:
The comment and `??` are not valid xml. Since we are populating the create date in the sql, we don't need a default value here. DAO calls should be populating this value on insert. Postgres has a `CURRENT_DATE` function, and [liquibase might also be able to handle this](https://stackoverflow.com/questions/9089678/liquibase-insert-insert-current-date) (caveat, have not tried this so it could be incorrect).